ros/processor/insights_engine_consumer.py

```diff
@@ -199,7 +210,10 @@ def process_report(self, host, platform_metadata, reports, system_metadata, perf
                     "state": SYSTEM_STATES[state_key],
                     "operating_system": system.operating_system,
                     'psi_enabled': system_metadata.get('psi_enabled'),
+                    'instance_type': suggested_instance_type,
+                    'instance_price': suggested_instance_price,
```

Comment:
Is it really required to separate out an instance type and its price? IMO there should be one column that would contain an array list of all the candidates and their respective prices.
ex: `candidates`: `[ [type1, price1], [type2, price2], ...]` 
Then we can easily get the first pair from the array as per our requirement. 
This arises as we also show the first three candidates in the recommendation section. 

Let me know your thoughts on this. Thanks!